src/space.cpp

```diff
@@ -120,6 +120,33 @@ const no_space_table_t no_space_table[] =
 };
 
 
+bool token_is_within_Trailing_Return(chunk_t *pc)
```

Comment:
BTW, how come some words are capitalized?

Reply:
Do we have a rule for that? Where?